IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoad/Entities/IfcOpenCrossProfileDef/DocEntity.xml

```diff
@@ -13,6 +13,9 @@
 		<DocAttribute Name="Tags" UniqueId="e0750c4d-1efe-493d-9699-706d8d0bc7d6" DefinedType="IfcLabel" AttributeFlags="1" AggregationType="1" AggregationLower="2" AggregationUpper="?">
 			<Documentation></Documentation>
 		</DocAttribute>
+		<DocAttribute Name="StartPoint" UniqueId="4f5b213a-261b-4c67-b80c-cb8b72473faa" DefinedType="IfcCartesianPoint">
+			<Documentation>The start point of the profile defined by a Cartesian point.</Documentation>
+		</DocAttribute>
```

Comment:
I'm not sure, but I would make this parameter OPTIONAL with the default value of (0,0).

Proposal of documentation improvement (discussable and please correct my English):

> The start point of the profile defined by a Cartesian point in profile's coordinate system. The provided slopes and widths emerge from this point. If not provided, the starting point is (0,0).

Reply:
I would actually prefer a mandatory. At least that way you know that the implementer actually wanted to set 0,0

Reply:
I'm fine with a mandatory as well. Then I'd revise the last sentence of my proposal to read 

> Usually, this would be (0,0), meaning that the profile starts in the coordinate origin of the profile.

Reply:
> Usually, this would be (0,0), meaning that the profile starts in the coordinate origin of the profile.

I would suggest it's actually unusual for the start point to be 0,0  The profile would normally not start on the directrix (usually the crown in a road).  And if it is offset in a vertical sense, then as soon as the profile is rotated to a vertical plane, then the profile won't intersect the directrix in at the same linear distance point.